src/main/pages/che-7/hands-on-guidance/proc_creating-a-workspace-by-importing-source-code-of-your-project.adoc

```diff
@@ -1,27 +1,111 @@
 [id="creating-a-workspace-by-importing-source-code-of-your-project_{context}"]
-= Creating a workspace by importing source code of your project
+= Creating a workspace by importing the source code of your project
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+This section is about how to create a new workspace to edit an existing codebase.
+
+There are 2 ways to do that *before* starting a workspace:
```

Comment:
To create a new workspace to edit  an existing codebase, use one of the following two methods *before* you start the workspace: